api/handler.go

```diff
@@ -0,0 +1,43 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/hashicorp/consul-terraform-sync/api/oapigen"
+	"github.com/hashicorp/consul-terraform-sync/logging"
+)
+
+// Make sure we conform to ServerInterface
+var _ oapigen.ServerInterface = (*Handlers)(nil)
+
+const (
+	handlerSubsystemName = "handler"
```

Comment:
Not sure how much this const is tied to `Handlers`. Does it make sense to change this to `handlersSubsystemName = "handlers"`?